plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/TestClickHouseTypeMapping.java

```diff
@@ -416,6 +415,19 @@ public void testDate(ZoneId sessionZone)
                 .execute(getQueryRunner(), session, clickhouseCreateAndInsert("tpch.test_date"));
     }
 
+    @Test
+    public void testUnsupportedDate()
+    {
+        try (TestTable table = new TestTable(getQueryRunner()::execute, "test_unsupported_date", "(dt date)")) {
+            assertQueryFails(
+                    format("INSERT INTO %s VALUES (DATE '1969-12-31')", table.getName()),
+                    "Epoch must be between 0 and 49710: -1");
```

Comment:
It would be better to format these as dates, since that is what the user interacts with. How Clickhouse stores dates internally shouldn't matter to the Trino user.

> Date must be between 1970-01-01 and 2106-02-07: 1969-12-31